pkg/controller/kms/key/setup.go

```diff
@@ -0,0 +1,256 @@
+package key
+
+import (
+	"context"
+	"sort"
+
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
I just saw, that dynamodb imports `aws` from `pkg/clients` instead. Is this necessary or is it fine to import the aws-sdk-go directly?

Reply:
I think most controllers use `pkg/clients` for those helper functions since it has some helpers that do not exist in aws-sdk-go that you might eventually need.

Reply:
changed